docs/contributors/run-based-logs.md

```
@@ -0,0 +1,155 @@
+# Run-Based Logs for Scheduled Tasks — Contributor Guide
+
+This guide documents how the scheduled-task observability feature (Runs tab in Backstage) is implemented on top of the upstream observability events pipeline introduced in v1.2.0-m.1.
+
+## Quick architecture
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)